chrome-extension/content.js

```diff
@@ -146,3 +200,18 @@ async function addItem() {
     // TODO: display error message on frontend
   }
 }
+
+/**
+ * Helper to return a list of selected options from a multi-select dropdown
+ * @param {String} dropdownElementId 
+ */
+function getSelectedOptions(dropdownElementId) {
+  const selectedCategories = [];
+  const categoryOptions = document.getElementById(dropdownElementId).options;
+  for(option of categoryOptions) {
```

Comment:
nit: Should this be `for (option of categoryOptions)`?

Reply:
yes thank you!